core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ResourceSearchRequest.java

```diff
@@ -66,5 +73,20 @@ protected Resources toCollectionResource(RepositoryContext context, Hits<Resourc
 	protected Resources createEmptyResult(int limit) {
 		return new Resources(limit, 0);
 	}
-
+	
+	@Override
+	protected void toQuerySortBy(RepositoryContext context, Builder sortBuilder, Sort sort) {
+		if (sort instanceof SortField) {
+			SortField sortField = (SortField) sort;
+			if (ResourceSearchRequestBuilder.TYPE_RANK.equals(sortField.getField())) {
```

Comment:
In the case of term-based custom sorting in `SnomedConceptSearchRequest`, we did not add the constant for the special sort field to the fields in `SnomedConcept` or `SnomedConceptDocument`, only to the search request builder, similar to this part. @cmark What do you think, where should these constants appear?

Reply:
The special `term` sort for SNOMED CT Concepts is actually registered only at REST API level which works but is not ideal, let's move that to the SnomedConcept.Fields.ALL constant and then it will be consistent with the other sort field declarations.

Reply:
Done